js/lib/ForceUnit.js

```diff
@@ -0,0 +1,218 @@
+                "kilonewton": "pound-force",
+                "meganewton": "pound-force",
+                "giganewton": "pound-force",
+                "dyne": "pound-force",
+                "kilogram-force": "pound-force"
+            },
+            "imperial": {
+                "millinewton": "pound-force",
+                "newton": "pound-force",
+                "kilonewton": "pound-force",
+                "meganewton": "pound-force",
+                "giganewton": "pound-force",
+                "dyne": "pound-force",
+                "kilogram-force": "pound-force"
+            }
+        },
+        "uscustomary": {
+            "metric": {
+                "poundal": "newton",
+                "pound-force": "newton"
+            }
+        },
+        "imperial": {
+            "uscustomary": {
+                "poundal": "newton",
+                "pound-force": "newton"
+            }
+        }
+    }
+};
+
+ForceUnit.aliases = {
+    "N": "newton",
+    "newtons": "newton",
+    "milli newtons": "millinewton",
+    "milli newton": "millinewton",
+    "milli-newton": "millinewton",
+    "mN": "millinewton",
+    "kilo newtons": "kilonewton",
+    "kilo newton": "kilonewton",
+    "kilo-newton": "kilonewton",
+    "kN": "kilonewton",
+    "mega newtons": "meganewton",
+    "mega newton": "meganewton",
+    "mega-newton": "meganewton",
+    "MN": "meganewton",
+    "giga newtons": "giganewton",
+    "giga newton": "giganewton",
+    "giga-newton": "giganewton",
+    "giga-newton": "giganewton",
+    "GN": "giganewton",
+    "dynes": "dyne",
+    "dyn": "dyne",
+    "kilopond": "kilogram-force",
+    "kiloponds": "kilogram-force",
+    "kilogram force": "kilogram-force",
+    "kilograms force": "kilogram-force",
+    "kilograms-force": "kilogram-force",
+    "kp": "kilogram-force",
+    "poundals": "poundal",
+    "pdl": "poundal",
+    "pound force": "pound-force",
+    "pounds force": "pound-force",
+    "lb force": "pound-force",
+    "pounds force": "pound-force",
+    "lbs force": "pound-force",
+    "lbf": "pound-force"
+};
+
+ForceUnit.aliasesLower = {};
+for (var a in ForceUnit.aliases) {
+    ForceUnit.aliasesLower[a.toLowerCase()] = ForceUnit.aliases[a];
+}
+
+/**
+ * Convert a force to another measure.
+ * @static
+ * @param to {string} unit to convert to
+ * @param from {string} unit to convert from
+ * @param force {number} amount to be convert
+ * @returns {number|undefined} the converted amount
+ */
+ForceUnit.convert = function(to, from, force) {
+    from = Measurement.getUnitIdCaseInsensitive(ForceUnit, from) || from;
+    to = Measurement.getUnitIdCaseInsensitive(ForceUnit, to) || to;
+    var fromRow = ForceUnit.ratios[from];
+    var toRow = ForceUnit.ratios[to];
+    if (typeof(from) === 'undefined' || typeof(to) === 'undefined') {
+        return undefined;
+    }
+    return force * fromRow[toRow[0]];
+};
+
+/**
+ * @private
+ * @static
+ */
+ForceUnit.getMeasures = function () {
```

Comment:
just thought: We have two very similar name functions `getMeasure` and `getMeasures` Wouldn't it confuse users?

Reply:
That is how the design is for all of the units. We are sort of stuck with it now. We can deprecate `getMeasures` and make it an alias for `getMeasuresList` I guess.

Reply:
Yeah, I checked all other unit features have the same functions. I agree to make an alias and deprecated getMearues